src/lib/jitter/jitter.h

```diff
@@ -0,0 +1,24 @@
+/*
+* Jitter utility functions interface
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/build.h>
+
+#ifdef BOTAN_HAS_JITTER
+extern "C" {
+   #include <jitterentropy.h>
+}
+
+namespace Botan {
+
+rand_data* jitter_collector_create();
+void jitter_collector_free(rand_data* collector);
+void jitter_buffer(rand_data* collector, std::span<uint8_t> buf);
+void jitter_buffer(rand_data* collector, uint8_t buf[], size_t len);
```

Comment:
The interface exposed by this should map to one of the existing types within the library

`EntropySource` - used to feed a userspace PRNG such as HMAC-DRBG
`RandomNumberGenerator` - generates outputs directly